.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["eslint:recommended", "next", "next/core-web-vitals", "prettier"]
```

Comment:
Not sure if we should use both next & next/core-web-vitals for eslint extensions